docs/api/range.md

```diff
@@ -52,7 +52,19 @@ TODO: Playground Example
 
 ## Dual Knobs
 
-TODO: Playground Example
+Dual knobs introduce two knob controls that users can use to select a value at a lower and upper bounds. When selected, the Range will emit an `ionChange` event with a value object, containing the upper and lower values selected.
+
+```js
+// Example value event detail
```

Comment:
Optional: We could also direct them to the `RangeValue` type: https://ionic-docs-git-range-playground-dual-knobs-ionic1.vercel.app/docs/api/range#rangevalue